ui.apps/src/main/content/META-INF/vault/filter.xml

```diff
@@ -2,6 +2,7 @@
 <workspaceFilter version="1.0">
     <filter root="/apps/frontify-aem-connector"/>
     <filter root="/apps/wcm" mode="merge"/>
+    <filter root="/apps/cq" mode="merge"/>
```

Comment:
You may want to limit this to /apps/cq/experience-fragments/content/editor.

Reply:
I could update the filter path to /apps/cq/experience-fragments

Any more entries in the path result in: 
Filter root's ancestor '/apps/cq/experience-fragments' is not covered by any of the specified dependencies.